scss/_forms.scss

```diff
@@ -28,7 +28,7 @@ textarea {
   vertical-align: middle;
   background-color: #fff;
   background-repeat: no-repeat; // Repeat and position set for form states (success, error, etc)
-  background-position: right center;
+  background-position: right 8px center;
```

Comment:
Ohhh yeah, wasn't thinking about that. Nice dude. Maybe add something there as a quick comment so folks don't think it's an error?